### ISSUE TITLE
symbol fonts need ActualText

### ISSUE THREAD

**Issue by mbertucci47** [latex3/tagging-project#272]
The [marvosym](https://www.ctan.org/pkg/marvosym) and [wasysym](https://www.ctan.org/pkg/wasysym) packages are marked as compatible, however shouldn't they (and any other text symbol font that doesn't map directly to unicode) need ActualText? Here's a quote from the [pdfa best practices guide](https://pdfa.org/wp-content/uploads/2019/06/TaggedPDFBestPracticeGuideSyntax.pdf):
> Where symbolic characters (such as Wingdings glyphs) are used, use ToUnicode table entries, or ActualText, where the ActualText would contain the appropriate Unicode. Where no Unicode code point is available, use appropriate text conveying the meaning within an ActualText attribute. The use of the Unicode PUA is discouraged, as no predefined meaning is associated with Unicode values in the PUA.

**Comment by u-fischer** [comment]
yes partially compatible would be better. They do not disturb tagging, but the missing correct ToUnicode is not pdf/UA.  But ActualText is probably not needed (or only in a few cases), all glyphs seems to have sensible names, and so if there is an unicode counterpart it should be possible to declare a mapping with `\pdfglyphtounicode`. 

Additionally wasysym errors if amsmath is loaded after it.

**Comment by cfr42** [comment]
Note: also need updated LuaHBTeX for LuaLaTeX support.

Example for `marvosym` to save somebody else having to duplicate one:

Note that it is the pdfLaTeX result which shows there is a problem with `marvosym` itself. The LuaLaTeX result is expected.

```latex
\DocumentMetadata
  {
    lang=en-US,
    pdfversion=2.0,
    pdfstandard=ua-2,
    tagging=on
  }

\documentclass{article}
\usepackage{marvosym}

\title{pifont tagging test}

\begin{document}

\FemaleFemale

\end{document}
```

For LuaLaTeX, we get
```xml
<PDF>
 <StructTreeRoot>
  <Document xmlns="http://iso.org/pdf2/ssn"
     id="ID.02"
    >
   <text-unit xmlns="https://www.latex-project.org/ns/dflt"
      id="ID.05"
      rolemaps-to="Part"
     >
    <text xmlns="https://www.latex-project.org/ns/dflt"
       id="ID.06"
       xmlns:Layout="http://iso.org/pdf/ssn/Layout"
       Layout:TextAlign="Justify"
       rolemaps-to="P"
      >
     <?MarkedContent page="1" ?>[TEXT]
    </text>
   </text-unit>
  </Document>
 </StructTreeRoot>
</PDF>
```
And for pdfLaTeX:
```xml
<PDF>
 <StructTreeRoot>
  <Document xmlns="http://iso.org/pdf2/ssn"
     id="ID.02"
    >
   <text-unit xmlns="https://www.latex-project.org/ns/dflt"
      id="ID.05"
      rolemaps-to="Part"
     >
    <text xmlns="https://www.latex-project.org/ns/dflt"
       id="ID.06"
       xmlns:Layout="http://iso.org/pdf/ssn/Layout"
       Layout:TextAlign="Justify"
       rolemaps-to="P"
      >
     <?MarkedContent page="1" ?>[TEXT]
    </text>
   </text-unit>
  </Document>
 </StructTreeRoot>
</PDF>
```
